api-reference/user-app/vitals/vital-information-table.mdx

```diff
@@ -0,0 +1,20 @@
+Here is the information regarding various vitals:
+
+| **Vital Name**                                    | **LOINC Code** | **Measurement Unit**      | **MCUM UCUM Code**   | **EKA ID**        |
+|:--------------------------------------------------|:---------------|:--------------------------|:----------------------|:-------------------|
+| **Min Heart Rate**                              |                | BPM (beats per minute)     |                      |                   |
+| **Max Heart Rate**                              |                |                           |                      |                   |
+| **Mean Heart Rate**                             |                |                           |                      |                   |
+| **Random Blood Glucose**                        | 74774-1        | mg/dL                     |                      | lb-7881503220     |
+| **Fasting Blood Glucose (overnight)**            | 1558-6         | mg/dL                     |                      | lb-5349691605     |
+| **After Food Blood Glucose (>2hr after food)**   | 1521-4         | mg/dL                     |                      | lb-5528009855     |
+| **Blood Oxygen**                                | 2708-6         | %                         |                      | lb-1771358750     |
+| **BMI**                                         | 39156-5        | kg/m2                     |                      | lb-5305097546     |
+| **Steps**                                       | 41950-7        | steps (absolute count)    |                      | lb-5279709676     |
+| **Stress**                                      |                |                           |                      |                   |
+| **Sleep**                                       |                | minutes (duration)        |                      |                   |
+| **Height**                                      | 8302-2         | cm                        |                      | lb-7561569471     |
+| **Weight**                                      |                | kg                        |                      |                   |
+| **Systolic BP**                                 |                | mm Hg                     |                      |                   |
+| **Diastolic BP**                                |                | mm Hg                     |                      |                   |
+| **Blood Pressure**                              |                |                           |                      |                   |
```

Comment:
Need to add a column too to describe the type of measured_at (datetime or period)

Reply:
Maintain this in notion as well pls, thats for internal reference and after that we'll update it here once we have supported it in code